src/config/dynamic-theme-fixes.config

```diff
@@ -14611,14 +14611,20 @@ CSS
 div[class*="bym"][role="navigation"], .afC, .agJ {
     background-color: var(--darkreader-neutral-background) !important;
 }
+.agJ:hover, .ag5 {
+    background-color: #181A1B !important;
```

Comment:
We can use light scheme in Dark Reader too and forcing dark colors in CSS can mess it a little bit.
To avoid this we use ${source_color} instead of inverted one.
So I think we should use here:
`background-color: ${#E8E6E5} !important;`

Reply:
Fixed